auth.go

```diff
@@ -8,13 +8,23 @@ import (
 	"log"
 	"net/http"
 	"os"
+	"strings"
 	"time"
 
 	"github.com/nlopes/slack"
 
 	log15 "gopkg.in/inconshreveable/log15.v2"
 )
 
+const (
```

Comment:
oops, sorry i missed that. I'd make this a type

```go
type Scope string

const (
    Webhook Scope = "webhook"
    // etc
)
```

and then have a `[]Scope` instead of `[]string`